introduction.md

```diff
@@ -0,0 +1,2 @@
+I'm Cyberguille Software Development oriented to Space Geophysics. Current projects are focused in Forecast Model and Data Mining of local ionospheric parameters.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.